lively.morphic/undo.js

```diff
@@ -159,6 +160,13 @@ export class UndoManager {
     return undo;
   }
 
+  removeLatestUndo () {
+    this.undoStop();
+    const undo = this.undos.pop();
+    arr.remove(this.grouping.current, undo);
+    return undo;
+  }
+
   undo () {
     this.undoStop();
```

Comment:
Can you replace these 2 lines by
` const undo = this.removeLatestUndo();`

Reply:
Just to make sure that I understand you correctly (I sometimes get really confused by the way GitHub formats these, sorry) - we are talking about the implementation of the `undo` method?
If so, will do!

Reply:
Yes!

Reply:
Done :)